### PR TITLE
Dtspb 1070 document controller tests

### PR DESCRIPTION
**Pull request by arun-hmcts**
**Before creating a pull request make sure that:**

- [X ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [X ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-1070

### Change description ###

Added test for the following endpoints in DocumentController
 "/document/assembleLetter";
 "/document/default-reprint-values";

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by mounikahmcts — APPROVED** [review]

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/documents/SolBaCcdServiceDocumentsTests.java [hunk 1]
```diff
@@ -800,17 +814,124 @@ public void verifyWillLodgementDepositReceiptShouldReturnOkResponseCode() {
     }
     
     //Commented out due to Docmosis not allowing screen readers as images overlay all text
```
Comment (vinceholt) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/documents/SolBaCcdServiceDocumentsTests.java:816]: as commented code is now removed, we can also remove this comment

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/printservice/SolCcdServicePrintServiceTests.java [hunk 2]
```diff
@@ -87,4 +90,25 @@ public void verifyPaTemplateDetails() {
...
+    public void verifyprobateManLegacyCaseReturnsBadResponseCode() {
+        Response response = RestAssured.given().relaxedHTTPSValidation()
+                .headers(utils.getHeadersWithUserId("serviceToken","userId"))
+                .when().get("/template/probateManLegacyCase");
+        response.prettyPrint();
+
+        assertThat(response.statusCode(),is(equalTo(403)));
+        assertTrue(response.getBody().asString().contains("Access Denied"));
+    }
 }
```
Comment (vinceholt) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/printservice/SolCcdServicePrintServiceTests.java]: carriage return required at end of file

File: src/functionalTest/resources/json/document/assembleLetterTransform.json [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+        }
+      ]
+    }
+  },
+  "boAssembleLetterSendToBulkPrint": "No",
+  "paperForm": "No",
+  "last_modified": null,
+  "id": 1542380211044655,
+  "grantSignatureBase64": null
+}
```
Comment (vinceholt) [inline src/functionalTest/resources/json/document/assembleLetterTransform.json]: carriage return at end of file

**Review by vinceholt — APPROVED** [review]
Approved in anticipation of couple of very minor things being resolved

**Pull request merged**